keps/sig-node/4622-topologymanager-max-allowable-numa-nodes/README.md

```diff
@@ -249,7 +244,7 @@ For GA:
 
 #### GA
 
-- Add a metrics: `kubelet_topology_manager_admission_time`.
+- Add a metric: `kubelet_topology_manager_admission_time`.
```

Comment:
please add a note documenting we can use an existing metric

Reply:
updated.

Reply:
We already have this metric in the kubelet code.